elastic/kibana
# [Discover] Reload route when Reset Search is clicked

## Summary
The **Reset search**  button of Discover button didn't work as expected, this PR triggers a reload of the route that's necessary to reset Discover's state to the last persisted version of the saved search.

![Discover-ResetSearch-btn](https://user-images.githubusercontent.com/17003240/81388909-ab2ca700-9121-11ea-9043-7dcd2ca3ca5e.png)

### How to test
1. Save the search
2. Apply some changes to query
3. Click the **Reset the Search** button
4. Changes to to query should be reset to the persisted state

Fixes #65821 

Note that a reload of the route shouldn't be necessary to restore the saved search, but will be removed in future deangularization steps, for sure.

### Checklist
